packages/mobile/src/components/composer-input/ComposerInput.tsx

```diff
@@ -353,14 +355,14 @@ export const ComposerInput = forwardRef(function ComposerInput(
 
       // Start user autocomplete
       if (key === AT_KEY && onAutocompleteChange) {
-        const cursorPosition = selection?.start ?? 0
+        const cursorPosition = selectionRef.current?.start ?? 0
         setAutocompletePosition(cursorPosition)
         setIsAutocompleteActive(true)
       }
 
-      if (key === BACKSPACE_KEY && selection) {
-        const textBeforeCursor = value.slice(0, selection.start)
-        const cursorPosition = selection.start
+      if (key === BACKSPACE_KEY && selectionRef.current) {
+        const textBeforeCursor = value.slice(0, selectionRef.current.start)
+        const cursorPosition = selectionRef.current.start
```

Comment:
same here. is there a reason we aren't reusing the a shared `cursorPosition` variable in all of these locations?

Reply:
nice. just find+replace. much cleaner ty